css/src/components/banner.scss

```diff
@@ -22,6 +22,12 @@ $banner-border-width: $border-width !default;
 	border-block: $banner-border-width solid $banner-border-color;
 	line-height: $line-height-normal;
 
+	&[data-dismissable] {
```

Comment:
classes for styling, and `data-` for behavior?

Reply:
perhaps this is a case when the `has` selector would work well? `*:has(.dismiss) {}`?

Reply:
@wibjorn We're still [waiting on Firefox support for `:has`](https://developer.mozilla.org/en-US/docs/Web/CSS/:has#browser_compatibility), I think.

Reply:
I don't have a better idea of how to target it, since as Ben mentioned, `:has` isn't supported in FF. I was trying to come up with some universal solution but it would really depend on the component's markup dismiss used within.

Reply:
In keeping with classes for styling, we'd need a class: `.dismissable`

Reply:
Updated. What do you think about using `<component>-dismissable` class instead? Might align better with practices.

Reply:
yeah that sounds good to me, good idea!